dynet/lstm.cc

```diff
@@ -485,8 +485,7 @@ Expression VanillaLSTMBuilder::add_input_impl(int prev, const Expression& x) {
     i_aot = pick_range(tmp, hid * 2, hid * 3);
     i_agt = pick_range(tmp, hid * 3, hid * 4);
     Expression i_it = logistic(i_ait);
-    // TODO(odashi): Should the forget bias be a hyperparameter?
-    Expression i_ft = logistic(i_aft + 1.f);
+    Expression i_ft = logistic(i_aft + forget_bias);
```

Comment:
There should be an `if` statement here to avoid the addition if `forget_bias` is zero. This will save a little bit of time and memory.

Reply:
Ok i will fix it asap

Reply:
Thanks. Also, you don't need to close the PR every time you make changes. Just push to the branch and your new changes will be reflected here.